### PR TITLE
Various small improvements in the Texinfo version of the manual

### PR DESCRIPTION
**Pull request by scymtym**

### CODE REVIEW

File: Documentation/Manual/Texinfo/chap-debugger.texi [hunk 1]
```diff
@@ -22,66 +22,74 @@ To get up and running quickly with Debugger:
...
+should be supplied by the Lisp implementation allowing to encapsulate or
 supply different debugger for recursive debugger calls.
 @end deffn
 
 @deffn {macro} {with-debugger} options &body body
 
-Wraps the code in @t{body} to invoke clim debugger when condition is
-signalled (binds @t{*debugger-hook*} to @t{#'debugger}). Bindings are
-inherited by new threads. @t{options} are not used at the moment.
+Executes the code in @var{body} invoking the Clim debugger when an
```
Comment (dkochmanski) [inline Documentation/Manual/Texinfo/chap-debugger.texi]: should be `CLIM` to preserve consistency

File: Documentation/Manual/Texinfo/extensions.texi [hunk 2]
```diff
@@ -227,78 +227,86 @@ T
...
+@anchor{Macro mcclim-raster-image:with-output-to-raster-image-stream}
+@anchor{Macro mcclim-raster-image:with-output-to-rgba-pattern}
+@deffn {Macro} {mcclim-raster-image:with-output-to-raster-image-stream} (stream-var stream format &rest options) &body body
+@deffnx {Macro} {mcclim-raster-image:with-output-to-rgba-pattern} (stream-var image &rest options) &body body
+@findex mcclim-raster-image:with-output-to-raster-image-stream
+@findex mcclim-raster-image:with-output-to-rgba-pattern
+Within @var{body}, @var{stream-var} is bound to a stream that produces a
+raster image.  This stream is suitable as a stream or medium argument to
+any CLIM output function, such as @code{draw-line*} or
+@code{write-string}.
```
Comment (dkochmanski) [inline Documentation/Manual/Texinfo/extensions.texi]: I would add warning here that the package will most likely change to clime in the future (or even comment whole section). These things are not well tested afaik and code needs cleanup.

Comment (dkochmanski) [reply Documentation/Manual/Texinfo/extensions.texi]: this remark applies to all macro exported from mcclim-raster-image:

File: Documentation/Manual/Texinfo/chap-debugger.texi (hunk 1 above)
Comment (scymtym) [reply Documentation/Manual/Texinfo/chap-debugger.texi]: I made macros for "CLIM" and "McCLIM" to ensure this.

**Comment by scymtym** [comment]
I can remove the Extensions commit for now.

**Comment by scymtym** [comment]
I just noticed that the Texinfo macros, when used in certain positions, break the LaTeX output.

**Comment by dkochmanski** [comment]
yes, I can confirm that with macros it doesn't build for me with texinfo 6.5.0

**Comment by scymtym** [comment]
I think i have made all requested changes. I removed the commit which improves the Extensions chapter for now.

**Pull request merged**